bokehjs/src/coffee/core/util/object.ts

```diff
@@ -27,6 +29,21 @@ export function clone<T>(obj: T): T {
   return extend({}, obj)
 }
 
+export function merge<T>(obj1: {[key: string] : Array<T>}, obj2: {[key: string]: Array<T>}): object {
```

Comment:
@mattpap I tried the type signature with obj1 and obj2 being objects, but there was some sort of warning. This seems to have fixed it, but I'm not certain it's the right thing to do. Also, I don't know if there is common notion of what "merge" should mean for objects. I guess this function has to take objects with arrays as values and unions those array values for two objects.